mk/gotest.sh

```diff
@@ -47,7 +46,7 @@ elif [[ $# -eq 2 ]]; then
   if [[ ${TEST,,} == bench* ]]; then
     RUN=$RUN' -bench '$TEST
```

Comment:
I didn't touch this and many other similar instances because it wasn't clear to me if `$TEST` and `$PKG` could contain wildcards that you _do_ want to expand (though if that's the case there may still be a problem because the vars are expanded multiple times...)